src/hooks/useBindInput.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+export interface BindInput {
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const useBindInput = (initialValue?: string, inputValidator?: (text: string) => boolean) => {
```

Comment:
```suggestion
const useBindInput = ({ initialValue, validator }: { initialValue?: string; validator?: (text: string) => boolean }) => {
```
현재는 `initialValue`를 반드시 인자로 넣어야 `validator`를 사용할 수 있는 구조인데, `validator`만 사용할 수도 있으면 더 편할 것 같아요!

Reply:
거기까지 생각을 못했네요! 동의합니당